app/src/main/java/fr/iut_rodez/pathpilot_android_client/util/VolleyErrorHandler.java

```diff
@@ -0,0 +1,123 @@
+package fr.iut_rodez.pathpilot_android_client.util;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.android.volley.AuthFailureError;
+import com.android.volley.NetworkError;
+import com.android.volley.NoConnectionError;
+import com.android.volley.ParseError;
+import com.android.volley.ServerError;
+import com.android.volley.TimeoutError;
+import com.android.volley.VolleyError;
+
+import java.nio.charset.StandardCharsets;
+
+import fr.iut_rodez.pathpilot_android_client.R;
+
+public class VolleyErrorHandler {
```

Comment:
Missing Javadoc comment to describe what the class is for

Reply:
Fix in `3f381088a74825447a65980cc8bb98e00dd60954`